### PR TITLE
Creating checkup-plugin-javascript, and moving the eslint-disable and outdated-deps task into it

### PR DESCRIPTION
**Pull request by carakessler**

### CODE REVIEW

File: packages/checkup-plugin-javascript/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# plugin-javascript
```
Comment (scalvert) [inline packages/checkup-plugin-javascript/README.md:1]:
```suggestion
# checkup-plugin-javascript
```

File: packages/checkup-plugin-javascript/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+An oclif plugin for Javascipt project tasks
```
Comment (scalvert) [inline packages/checkup-plugin-javascript/README.md:3]:
```suggestion
An checkup plugin for Javascipt project tasks
```

File: packages/checkup-plugin-javascript/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+<!-- toc -->
+
+- [plugin-javascript](#plugin-javascript)
```
Comment (scalvert) [inline packages/checkup-plugin-javascript/README.md:7]:
```suggestion
- [checkup-plugin-javascript](#checkup-plugin-javascript)
```

File: packages/checkup-plugin-javascript/README.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+```sh-session
+$ npm install -g checkup-plugin-javascript
+$ checkup-plugin-javascript COMMAND
+running command...
+$ checkup-plugin-javascript (-v|--version|version)
+checkup-plugin-javascript/0.0.1 darwin-x64 node-v12.16.1
+$ checkup-plugin-javascript --help [COMMAND]
+USAGE
+  $ checkup-plugin-javascript COMMAND
+...
```
Comment (scalvert) [inline packages/checkup-plugin-javascript/README.md:25]: We should consider removing this. We don't have commands in plugins. I think our generator needs to remove this too.

File: packages/checkup-plugin-javascript/README.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+```
+
+<!-- usagestop -->
+
+# Commands
+
+<!-- commands -->
+
+<!-- commandsstop -->
```
Comment (scalvert) [inline packages/checkup-plugin-javascript/README.md:34]: pew pew 🔫

File: packages/checkup-plugin-javascript/package.json [hunk 6]
```diff
@@ -0,0 +1,55 @@
+{
+  "name": "checkup-plugin-javascript",
+  "description": "An oclif plugin for Javascript project tasks",
+  "version": "0.0.14",
+  "author": "Cara Kessler @carakessler",
+  "bugs": "https://github.com/checkupjs/checkup/issues",
+  "dependencies": {
+    "@checkup/core": "0.0.14",
```
Comment (scalvert) [inline packages/checkup-plugin-javascript/package.json:8]: We should float this so we automatically pick up updates.

File: packages/checkup-plugin-javascript/package.json [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+    "@oclif/command": "^1",
+    "@oclif/config": "^1",
+    "ember-template-recast": "^4.1.4",
+    "tslib": "^2"
+  },
+  "devDependencies": {
+    "@checkup/test-helpers": "0.0.14",
```
Comment (scalvert) [inline packages/checkup-plugin-javascript/package.json:15]: We should float this so we automatically pick up updates.

File: packages/checkup-plugin-javascript/package.json [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": "https://github.com/checkupjs/checkup",
+  "scripts": {
+    "build": "tsc --build",
+    "postpack": "rm -f oclif.manifest.json",
+    "prepack": "tsc --build && oclif-dev manifest && oclif-dev readme",
+    "test": "jest",
+    "version": "oclif-dev readme && git add README.md"
```
Comment (scalvert) [inline packages/checkup-plugin-javascript/package.json:51]: It's the oclif readme stuff that updates the readme with the commands section et al. This is what we should remove. We should likely do the same with our other plugins.

File: packages/cli/package.json [hunk 9]
```diff
@@ -19,6 +19,7 @@
     "@types/sloc": "^0.2.0",
     "chalk": "^4.0.0",
     "checkup-plugin-ember": "0.0.14",
+    "checkup-plugin-javascript": "0.0.14",
     "checkup-plugin-ember-octane": "0.0.14",
```
Comment (scalvert) [inline packages/cli/package.json:23]: I think we should float all of these dependencies.

File: packages/checkup-plugin-javascript/README.md (hunk 4 above)
Comment (carakessler) [reply packages/checkup-plugin-javascript/README.md:25]: ya agreed - i think there isnt rly anything that needs to be in the README aside from 
```
# checkup-plugin-javascript

An checkup plugin for Javascipt project tasks
```

**Pull request merged**